### PR TITLE
added ques in html css js and ds-algo

### PR DESCRIPTION
**Pull request by sharma2514**

### CODE REVIEW

File: css/answers/gopal-q10.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                <h4>
+                    <h4>Which command is used for the selection of all the elements of a paragraph?</h4>
+                </h4>
+            </div>
+            <br>
+            <br>
+            <div>
+                <strong>Answer:</strong> 
+                <p>
+                    The p[lang] command is used for selecting all the elements of a paragraph.
```
Comment (dakshkhetan) [inline css/answers/gopal-q10.html:52]: Please give an example or code snippet to demonstrate.

File: ds-and-algo/answers/gopal-q29.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                </h4>
+            <br>
+            <br>
+            <div>
+                <strong>Answer:</strong> 
+                <p>
+                    If inorder traversal of a binary tree is sorted, then the binary tree is BST. 
+                    The idea is to simply do inorder traversal and while traversing keep track of previous key value. 
+                    If current key value is greater, then continue, else return false.
+                </p>
```
Comment (dakshkhetan) [inline ds-and-algo/answers/gopal-q29.html:67]: Provide the code too.

**Review by dakshkhetan — CHANGES_REQUESTED** [review]
Please see the requested changes. :)

File: ds-and-algo/answers/gopal-q29.html (hunk 2 above)
Comment (sharma2514) [reply ds-and-algo/answers/gopal-q29.html:67]: i should send u code or i should send u pull request again?

Comment (sharma2514) [reply ds-and-algo/answers/gopal-q29.html:67]: private void inOrder(TreeNode node) {
    if (node == null) {
      return;
    }

    inOrder(node.left);
    System.out.printf("%s ", node.data);
    inOrder(node.right);
}

Comment (dakshkhetan) [reply ds-and-algo/answers/gopal-q29.html:67]: No, just create another commit in this branch locally and then push the changes.

File: css/answers/gopal-q10.html (hunk 1 above)
Comment (sharma2514) [reply css/answers/gopal-q10.html:52]: <!DOCTYPE html>
<html>
<head>
<style>
p:lang(it) { 
  background: yellow;
}
</style>
</head>
<body>

<p>I live in Italy.</p>
<p lang="it">Ciao bella!</p>

<p><b>Note:</b> For :lang to work in IE8, a DOCTYPE must be declared.</p>

</body>
</html>

Comment (dakshkhetan) [reply css/answers/gopal-q10.html:52]: Please add this in the file itself from your editor (like you did before).

**Comment by sharma2514** [comment]
sir i have done git push origin news-myques
anything else to do?

**Comment by sharma2514** [comment]
means i have to send PR again?

**Comment by dakshkhetan** [comment]
No, good job!

**Review by dakshkhetan — APPROVED** [review]

**Pull request merged**